pkgs/development/ruby-modules/bundled-common/default.nix

```diff
@@ -21,6 +21,7 @@
 , groups ? null
 , ignoreCollisions ? false
 , buildInputs ? []
+, additionalConfPaths ? []
```

Comment:
what do you think of using `extraConfPaths` instead?

Reply:
Sure, any reasoning just out of curiosity?

Reply:
I believe that the `extra` prefix is more common in nixpkgs

Reply:
Do you think `conf` is appropriate here (due to the derivation name), or should it be `config`?

Reply:
Let's go with `extraConfigPaths` and get this merged asap.